library/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@irontec/ivoz-ui",
-  "version": "1.5.1",
+  "version": "1.6.1",
```

Comment:
I don't see a 1.6.0 tag

Not sure this deserves a minor, but that's up to you.

Reply:
my fault, but yes, it deserves a minor, X.Y.Z 

X = breaking change
Y = features (This is a feature)
Z = Bug fixing